src/main/java/com/codeborne/selenide/WebDriverConditions.java

```diff
@@ -72,4 +74,16 @@ public static ObjectCondition<WebDriver> numberOfWindows(int numberOfWindows) {
   public static ObjectCondition<WebDriver> title(String expectedTitle) {
     return new Title(expectedTitle);
   }
+
+  @CheckReturnValue
+  @Nonnull
+  public static ObjectCondition<WebDriver> cookieWithName(String name) {
```

Comment:
Maybe just `cookie` and `cookieValue`?
All of because `name` and `value` parameters obviously describe method

Reply:
@adorne I agree that having one name `cookie` would be simpler:

* `shouldHave(cookie("session_id"));`
* `shouldHave(cookie("session_id", "9182827334645"));`

and maybe in future something like this:
* `shouldHave(cookie(new Cookie("session_id", "9182827334645")));`

Then it stays consistent e.g. with current attribute checks:
* `$.shouldHave(attribute("name"));`
* `$.shouldHave(attribute("name", "value"));`

Reply:
Okay, I agree with the consistency argument, I'll fix it.

Reply:
@asolntsev fixed.